arbitrum-docs/stylus/stylus-gentle-introduction.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'A gentle introduction to Stylus'
+sidebar_label: 'A gentle introduction to Stylus'
+description: ''
+author: amarrazza
+sme: amarrazza
+target_audience: todo
+sidebar_position: 1
+---
+
+# A gentle introduction: Stylus
+
+import PublicPreviewBannerPartial from './partials/_stylus-public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+This introduction is for developers who are interested in learning about Stylus, a new way to write smart contracts in popular programming languages, like Rust, alongside the EVM.
+
+### In a nutshell:
+
+- Stylus lets you write smart contracts in programming languages that compile down to WASM, such as **Rust, C, C++, and many others**.
+- Rich language support already exists for Rust: developers can use the Stylus SDK and CLI tool to **start building today.**
+- Arbitrum's full **EVM equivalence would be maintained** while adding a second, coequal WASM virtual machine.
+- Due to the superior efficiency of WASM programs, Stylus contracts are over **an order of magnitude faster with significantly lower gas fees**.
+- **Memory is 100-500x cheaper** in Stylus, enabling new use cases now that consuming RAM is viable on the blockchain.
```

Comment:
```suggestion
- **Memory is 100-500x cheaper** when using Stylus; this unlocks new use-cases that were previously too expensive to implement on-chain.
```

definitely a nit, feel free to discard any of these suggestions that don't strike you as "pure wins"

Reply:
consuming RAM gets at a specific pain point solidity devs experience, so I would vote to keep that part in